COMMITTERS.md

```diff
@@ -0,0 +1,90 @@
+#Committing changes to SDL
+
+We would like to make it easier for community members to contribute to SDL
+using pull requests. This makes the process of contributing a little easier for the contributor since they don't
+need to concern themselves with the question, "What branch do I base my changes
+on?"  This is already called out in the CONTRIBUTING.md.
+
+##Terminology
+Many of these terms have more than one meaning. For the purposes of this
+document, the following terms refer to specific things.
+
+**contributor** - A person who makes a change to SDL and submits a change
+set in the form of a pull request.
+
+**reviewer** - A person responsible for reviewing a pull request.
+
+**master branch** - [The base branch](https://github.com/smartdevicelink/sdl_core/tree/master).
+
+**develop branch** - The branch where bug fixes against the latest release or

```

Comment:
if there is a link to the master branch, I think it's logical to have link for the develop